client/src/components/UI/Button.tsx

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from "styled-components";
+import posed from "react-pose";
+
+interface Props {
+  readonly size?: string;
+  readonly rounded?: boolean;
+  readonly primary?: boolean;
+}
+
+export const Button = styled.button<Props>`
+  border: none;
+  font-size: 0.9rem;
+  padding: 10px 20px;
+  margin: 10px;
+  border-radius: ${props => (props.rounded ? "10px" : "none")};
```

Comment:
You can destructurize `round` props

`
border-radius: ${({ rounded }) => rounded ? '10px' : 'none'};
`